docs/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md

```diff
@@ -0,0 +1,642 @@
+            "id": 1,
+            "attributes": {
+              "name": "My category"
+            },
+            "meta": {
+              "availableLocales": []
+            }
+          }
+        ]
+      },
+      "meta": {
+        "availableLocales": []
+      }
+    },
+  ],
+  "meta": {}
+}
+
+```
+
+:::
+
+::::
+
+### Get an entry
+
+Returns an entry by id.
+
+:::: api-call
+
+::: request Example request
+
+`GET http://localhost:1337/api/restaurants/1`
+
+:::
+
+::: response Example response
+
+<!-- TODO: update with FoodAdvisor? -->
+```json
+{
+  "data": {
+    "id": 1,
+    "attributes": {
+      "title": "Restaurant A",
+      "description": "Restaurant A's description",
+      "categories": [
+        {
+          "id": 1,
+          "attributes": {
+            "name": "My category"
+          },
+          "meta": {
+            "availableLocales": []
+          }
+        }
+      ]
+    },
+    "meta": {
+      "availableLocales": []
+    }
+  },
+  "meta": {}
+}
+
+```
+
+:::
+
+::::
+
+### Create an entry
+
+Creates an entry and returns its value.
+
+If the [Internationalization (i18n) plugin](/developer-docs/latest/development/plugins/i18n.md) is installed, it's possible to use POST requests to the Content API to [create localized entities](/developer-docs/latest/development/plugins/i18n.md#creating-a-new-localized-entry).
+
+:::: api-call
+
+::: request Example request
+
+`POST http://localhost:1337/api/restaurants`
+
+<!-- TODO: update with FoodAdvisor? -->
+```json
+{
+  "data" {
+    "title": "Hello",
+    "relation": 2,
+    "relations": [2, 4],
+    "link": {
+      "id": 1,
+      "type": "abc"
+    }
+  }
+}
+```
+
+:::
+
+::: response Example response
+
+```json
+{
+  "data" {
+    "id": 1,
+    "attributes": {},
+    "meta": {}
+  },
+  "meta": {}
+}
+```
+
+:::
+
+::::
+
+### Update an entry
+
+Partially updates an entry by `id` and returns its value.
+Fields that aren't sent in the query are not changed in the database. Send a `null` value if you want to clear them.
+
+:::note
+It's currently not possible to [update the locale of an entry](/developer-docs/latest/development/plugins/i18n.md#updating-an-entry).
+:::
+
+:::: api-call
+
+::: request Example request
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+<!-- TODO: update with FoodAdvisor? -->
+```json
+{
+  "data" {
+    "title": "Hello",
+    "relation": 2,
+    "relations": [2, 4],
+  }
+}
+```
+
+:::
+
+::: response Example response
+
+```json
+{
+  "data" {
+    "id": 1,
+    "attributes": {},
+    "meta": {}
+  },
+  "meta": {}
+}
+```
+
+:::
+
+::::
+
+### Delete an entry
+
+Deletes an entry by id and returns its value.
+
+:::: api-call
+
+::: request Example request
+
+`DELETE http://localhost:1337/api/restaurants/1`
+
+:::
+
+::: response Example response
+
+<!-- TODO: update with FoodAdvisor? -->
+```json
+{
+  "data" {
+    "id": 1,
+    "attributes": {},
+    "meta": {}
+  },
+  "meta": {}
+}
+```
+
+:::
+::::
+
+## API Parameters
```

Comment:
We could document that our query parameters use the LHS bracket syntax. 

This doc explain the pros and cons too if you want https://www.moesif.com/blog/technical/api-design/REST-API-Design-Filtering-Sorting-and-Pagination/